src/ussd_public.js

```diff
@@ -744,13 +769,20 @@ go.app = function() {
                     new Choice('not-satisfied', $('Not Satisfied')),
                     new Choice('very-unsatisfied', $('Very unsatisfied'))
                 ],
-                next: 'state_end_servicerating'
+                next: function(choice) {
+                    return go.utils_project
+                        .post_servicerating_feedback(self.im, q_id, q_text_en.args[0], choice.label, choice.value, 1, self.im.user.answers.invite_uuid)
+                        .then(function() {
+                            return 'state_end_servicerating';
+                        });
+                }
             });
         });
 
-        // EndState
+        // EndState 6
         self.add('state_end_servicerating', function(name) {
-            // set servicerating_unanswered flag to false...
+            // sets servicerating_unanswered to false indicating completed servicerating feedback
+            self.im.user.set_answer('servicerating_unanswered', false);
```

Comment:
This is not needed now, but we do need to do a PATCH to the `invite` to set `completed` to `true`.